src/Symfony/Component/ErrorCatcher/ErrorRenderer/XmlErrorRenderer.php

```diff
@@ -65,7 +66,7 @@ public function render(FlattenException $exception): string
         return <<<EOF
 <?xml version="1.0" encoding="{$this->charset}" ?>
 <problem xmlns="urn:ietf:rfc:7807">
-    <title>{$exception->getTitle()}</title>
+    <title>{$title}</title>
     <status>{$exception->getStatusCode()}</status>
```

Comment:
let's do the same for the status code and the charset, better be extra safe
maybe also check that this is done in other formats?